package.json

```diff
@@ -32,6 +32,7 @@
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-fetch-options": "^0.0.5",
+    "eslint-plugin-html": "^6.0.2",
```

Comment:
Interesting. Will this lint .html/.md files by default, or do you need to specify the additional file extensions to lint above on L10 in your `eslint src` command?

https://github.com/BenoitZugmeyer/eslint-plugin-html#no-file-linted-when-running-eslint-on-a-directory

Reply:
A bit hacky, but confirmed with using `DEBUG=eslint:cli-engine` and shimming the `--ext=.html,.js` to force linting of both .html and .js files. Without explicitly adding `.html` extension, it was only linting the JavaScript files.

```diff
DEBUG=eslint:cli-engine npm run lint -- --ext=.html,.js

> secure-proxy@1.0.0 lint /Volumes/Dev/github/mozilla/secure-proxy-2
> eslint src "--ext=.html,.js"

  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/component.js +0ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/connection.js +174ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/connectivity.js +28ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/constants.js +14ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/external.js +7ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/fxa.js +18ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/ipinfo.js +99ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/logger.js +16ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/main.js +11ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/messageService.js +75ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/network.js +28ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/offline.js +56ms
+ eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/page.html +7ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/proxyDownChecker.js +2ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/proxyStateObserver.js +18ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/storageUtils.js +4ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/survey.js +8ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/telemetry.js +7ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/ui.js +19ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/background/wellKnownData.js +25ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/commons/template.js +16ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/commons/utils.js +15ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/content/fetch-fxa-flow-params.js +19ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/experiments/proxyutils/api.js +6ms
+ eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/pages/welcome.html +53ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/pages/welcome.js +2ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/animations.js +5ms
+ eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/popup.html +7ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/popup.js +1ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/view.js +20ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/connecting.js +19ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/deviceLimit.js +6ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/disabled.js +8ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/loading.js +8ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/login.js +4ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/main.js +7ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/onboarding.js +5ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/paymentRequired.js +6ms
  eslint:cli-engine Lint /Volumes/Dev/github/mozilla/secure-proxy-2/src/popup/views/settings.js +6ms
  eslint:cli-engine Linting complete in: 972ms +9ms
```

I struggled to figure out how to count lines w/ piping to grep, but this seems to work by piping stderr to stdout before piping through `grep` and `wc -l`:

```sh
DEBUG=eslint:cli-engine npm run lint -- --ext=.html,.js 2>&1 | grep "Lint /Volumes/" | wc -l
```

Reply:
Nice. So far I'm mainly interested in the JS files. But covering HTML sounds like a good idea.